frontend/public/locales/en-US/pipeline.json

```diff
@@ -410,6 +410,8 @@
   },
   "upgrade": {
     "title": "Upgrade Pipeline",
-    "tip": "Are you sure you want to upgrade the pipeline to this version?"
+    "tip": "Are you sure you want to upgrade the pipeline to this version?",
+    "needTimezone": "Please configure the timezone of the following applications first, and then return to the current page to upgrade.",
```

Comment:
To upgrade the pipeline, please specify a reporting time zone for the app(s) registered in this pipeline.